tests/parser/functions/test_default_parameters.py

```diff
@@ -190,7 +190,7 @@ def bar() -> uint256(wei):
     c.foo(transact={'value': 31337})
     assert c.bar() == 31337
     c.foo(666, transact={'value': 9001})
-    assert c.bar() == 666
+    assert c.bar() == 31337 + 666
```

Comment:
Maybe also add a line that says "balance of `c` contract is `31337 + 9001`" to show the difference in using default args.